Common/Data/Custom/TradingEconomics/TradingEconomicsCalendar.cs

```diff
@@ -150,7 +150,7 @@ public class TradingEconomicsCalendar : BaseData
         /// Unique symbol used by Trading Economics
         /// </summary>
         [JsonProperty(PropertyName = "Symbol")]
-        public string Symbol { get; set; }
+        public string TESymbol { get; set; }
```

Comment:
In LEAN parlance, can call this `Ticker` as it's just the `string` and not a full `Symbol` object.